editor/src/components/editor/variablesmenu.spec.browser2.tsx

```diff
@@ -106,6 +112,59 @@ describe('variables menu', () => {
     })
   })
 
+  describe('props', () => {
+    it('shows and inserts props in scope', async () => {
+      const editor = await renderTestEditorWithProjectContent(
+        makeTestProjectContents(),
+        'await-first-dom-report',
+      )
+
+      await selectComponentsForTest(editor, [
+        EP.fromString(`${BakedInStoryboardUID}/${TestSceneUID}/${TestAppUID}:container`),
+      ])
+
+      await openVariablesMenu(editor)
+
+      document.execCommand('insertText', false, 'imgProp')
+      expect(getInsertItems().length).toEqual(1)
+      expect(getInsertItems()[0].innerText).toEqual('imgProp')
+
+      const filterBox = await screen.findByTestId(InsertMenuFilterTestId)
+      forceNotNull('the filter box must not be null', filterBox)
+
+      await act(async () => {
+        fireEvent.keyDown(filterBox, { key: 'Enter', keycode: 13 })
+      })
```

Comment:
We have a bunch of helper functions for firing events in `event-helpers.test-utils.tsx`, including one for `pressKey`: https://github.com/concrete-utopia/utopia/blob/master/editor/src/components/canvas/event-helpers.test-utils.tsx#L749 - If that function isn't suitable as it stands, please do tweak it to support this use case, since we want to avoid manually firing the events like this as those helpers exist to ensure that we can better simulate a real interaction by calling the combination of events that would be fired in the right order